packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -431,20 +413,20 @@ function* getAudioPurchaseInfo({
       'finalized'
     )
 
-    const estimatedLamports = BN.max(
-      new BN(inSol)
-        .add(new BN(associatedAccountCreationFees))
-        .add(new BN(transactionFees))
-        .add(new BN(rootAccountMinBalance))
-        .sub(new BN(existingBalance)),
-      new BN(0)
+    const estimatedLamports = Math.max(
```

Comment:
This could burn us - `Math.max` doesn't support `BigInt` so if numbers get too large for converting to `Number` this will throw. Comparing to 0 might be safer here

Reply:
i think these are all `number`s. but were you thinking of something like this?
```
    const estimatedLamportsNumber =
      inSol +
      associatedAccountCreationFees +
      transactionFees +
      rootAccountMinBalance +
      existingBalance
    const estimatedLamports =
      estimatedLamportsNumber > 0 ? estimatedLamportsNumber : 0
```

Reply:
Ah nvm if they're all numbers! worth checking other spots though